moveit_core/collision_detection/src/allvalid/collision_env_allvalid.cpp

```diff
@@ -97,7 +97,7 @@ void collision_detection::CollisionEnvAllValid::checkRobotCollision(const Collis
 }
 
 void collision_detection::CollisionEnvAllValid::distanceRobot(const collision_detection::DistanceRequest& req,
-                                                              collision_detection::DistanceResult& res,
+                                                              DistanceResult& res,
```

Comment:
Why is this only applied to the result and not to the request...?

Reply:
idk, I'm rather confused by clang-tidy.  I created this diff following the instructions on moveit.ros.org for running it in a loop.

Reply:
I think there is some weird mismatch going on with the fcl namespace (which has all the same types as `collision_detection/collision_common.h`. The fcl package has a compat header that attempts to remap all the types, to me it looks like clang-tidy runs into issues resolving all those. I suppressed the weird "DistanceResultsData" renaming for now and added an issue for it (#267).